packages/extensions/cloud/n8n-extension.manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "https://raw.githubusercontent.com/n8n-io/n8n/refs/heads/master/packages/%40n8n/extension-sdk/schema.json",
+	"name": "cloud",
+	"publisher": "n8n",
+	"displayName": "Cloud",
+	"description": "Extension for n8n Cloud",
+	"version": "1.0.0",
```

Comment:
No versioning for internal modules as they're tied to the monorepo release cycle?

Reply:
Good point! Removed version.